ci/docker/runtime_functions.sh

```diff
@@ -1391,6 +1391,15 @@ nightly_test_installation() {
     source ./tests/jenkins/run_test_installation_docs.sh docs/install/index.md 1 1686; ${1}
 }
 
+# Runs Imagenet inference
+nightly_test_imagenet_inference() {
+    set -ex
+    echo $PWD
+    cp /work/mxnet/build/cpp-package/example/imagenet_inference .
```

Comment:
Is this tested?

Reply:
yes. on ci dev. it exists.

Reply:
Spoke prematurely.
http://jenkins.mxnet-ci-dev.amazon-ml.com/blue/organizations/jenkins/NightlyTestsForBinaries/detail/move_imagenet_inference_nightly/20/pipeline/72

While it passed the `cp ...` line, 

Failed at this line in 
`./imagenet_inference --symbol_file ./model/Inception-BN-symbol.json --params_file ./model/Inception-BN-0126.params --dataset ./data/val_256_q90.rec --rgb_mean '123.68 116.779 103.939' --batch_size 1 --num_skipped_batches 50 --num_inference_batches 500`

Error being
```
./imagenet_inference: error while loading shared libraries: libmxnet.so: cannot open shared object file: No such file or directory
```

Unable to understand despite me adding 

https://github.com/ChaiBapchya/incubator-mxnet/blob/daa88e6ff8f4e98742ea46bbe77085147bf2aeca/tests/nightly/JenkinsfileForBinaries#L23